intercom-java/src/main/java/io/intercom/api/Conversation.java

```diff
@@ -208,6 +213,9 @@ private static boolean isAdminQuery(Map<String, String> params) {
     @JsonProperty("read")
     private boolean read;
 
+    @JsonProperty("state")
+    private String state;
```

Comment:
What's that property used for in the context of snoozing? There's no setter, is that expected?

Reply:
I think this maps to conversation state (open, closed, snoozed). We also have `open` )
which is boolean but that no longer fits with `snoozed` state.

https://developers.intercom.com/intercom-api-reference/reference#conversation-model

Reply:
Make sense :+1: